src/models/curriculum/problem.ts

```diff
@@ -40,11 +40,15 @@ const ModernProblemModel = types
     }
```

Comment:
Github isn't letting me post a comment on the actual line for some reason, but the import on line 3 is no longer used.